modules/xmpp/ResumeTask.js

```diff
@@ -49,7 +49,9 @@ export default class ResumeTask {
         this._cancelResume();
 
         this._resumeRetryN += 1;
-
+        if (this._networkOnlineListener) {
```

Comment:
Rather than here, I think the code currently in cancel should move to `_cancelResume` so it applies in both cases.

Reply:
Agree. Applied MR suggestion.